docs/tutorials/using-jsx-in-apostrophe.md

```diff
@@ -441,15 +467,16 @@ const CityComponent = (props) => {
       <SearchBox onSubmit={fetchWeather}>
         <input
           onChange={(e) => updateCity(e.target.value)}
-          placeholder="City"
+          placeholder='City'
         />
-        <button type={"submit"}>Search</button>
+        <button type={'submit'}>Search</button>
```

Comment:
It sounds like the curlies aren't adding any value here

Reply:
I think I had refactored that from using a variable and forgot to remove the curlies. Thanks.